changelog.d/5570.bugfix

```diff
@@ -0,0 +1 @@
+[MergedRoomCreationItem] DM: RoomDescriptionText use the roomName instead of memberName
```

Comment:
Use a more friendly message like:
"Use member name instead of room name in DM creation item"